pkg/flare/archive.go

```diff
@@ -393,30 +393,14 @@ func zipConfigFiles(tempDir, hostname string, confSearchPaths SearchPaths, perms
 	if config.Datadog.ConfigFileUsed() != "" {
```

Comment:
What if no config file is used for datadog.yaml but the system-probe is configured ?

Reply:
Then it won't take the system-probe config file into archive either. As I mentioned in the PR description, the only way this would work is if system-probe config is side by side with datadog.yaml because datadog-agent doesn't recognize `system-probe.yaml` in anyway. But I'm open to any better suggestions.

Reply:
I think this means we won't pickup the file in a containerize environment where we use env var. Are we OK with that ?

Reply:
Actually this might not work, as in container environment system-probe is never in the same container as datadog-agent, so they will always have different env vars. I don't think there's a good way to solve this.